### PR TITLE
react-application-nav-search-driven Upgrade to SPFx 1.20.0

### PR DESCRIPTION
**Pull request by dgirase**
> By submitting this pull request, you agree to the [contribution guidelines](https://github.com/pnp/sp-dev-fx-extensions/blob/main/CONTRIBUTING.md)

> If you aren't familiar with how to contribute to open-source repositories using GitHub, or if you find the instructions on this page confusing, [sign up](https://forms.office.com/Pages/ResponsePage.aspx?id=KtIy2vgLW0SOgZbwvQuRaXDXyCl9DkBHq4A2OG7uLpdUREZVRDVYUUJLT1VNRDM4SjhGMlpUNzBORy4u) for one of our [Sharing is Caring](https://pnp.github.io/sharing-is-caring/#pnp-sic-events) events. It's completely free, and we'll guide you through the process.

|        Q        |                    A                    |
| --------------- | --------------------------------------- |
| Bug fix/Update?        | yes                               |
| New feature?    | no                               |
| New sample?     | no                             |
| Related issues? | fixes #X, partially #Y, mentioned in #Z |

### CODE REVIEW

File: samples/react-application-nav-search-driven/package.json [hunk 1]
```diff
@@ -13,29 +13,29 @@
...
-    "@microsoft/sp-adaptive-card-extension-base": "1.18.2",
-    "@microsoft/sp-application-base": "1.18.2",
-    "@microsoft/sp-core-library": "1.18.2",
-    "@microsoft/sp-dialog": "1.18.2",
+    "@microsoft/sp-adaptive-card-extension-base": "1.20.0",
+    "@microsoft/sp-application-base": "1.20.0",
+    "@microsoft/sp-core-library": "1.20.0",
+    "@microsoft/sp-dialog": "1.20.0",
     "@types/react": "17.0.45",
     "@types/react-dom": "17.0.17",
```
Comment (Adam-it) [inline samples/react-application-nav-search-driven/package.json:23]: this was a mistake. all types should be a devDependency. Lets move it as well

File: samples/react-application-nav-search-driven/package.json [hunk 2]
```diff
@@ -13,29 +13,29 @@
...
     "react-dom": "17.0.1",
     "tslib": "2.3.1"
   },
   "devDependencies": {
-    "@microsoft/eslint-config-spfx": "1.18.2",
-    "@microsoft/eslint-plugin-spfx": "1.18.2",
+    "@microsoft/eslint-config-spfx": "1.20.2",
+    "@microsoft/eslint-plugin-spfx": "1.20.2",
     "@microsoft/rush-stack-compiler-4.5": "0.5.0",
     "@microsoft/rush-stack-compiler-4.7": "0.1.0",
```
Comment (Adam-it) [inline samples/react-application-nav-search-driven/package.json:32]: we should leave only the rush stack 4.7. We may remove the older one

**Review by Adam-it — CHANGES_REQUESTED** [review]
@dgirase Awesome work 👏👏👏
We are really close and I only notes some few minor things we should sort out before we merge. Please do give the a double check 🙏

Also we should update the `assets/sample.json` file. We should update the `updateDateTime` and `SPFX-VERSION` metadata in this file

File: samples/react-application-nav-search-driven/package.json (hunk 1 above)
Comment (dgirase) [reply samples/react-application-nav-search-driven/package.json]: I have update the devDependency

File: samples/react-application-nav-search-driven/package.json (hunk 2 above)
Comment (dgirase) [reply samples/react-application-nav-search-driven/package.json]: I have removed the older version of rush stack. Please check lastest commit

**Comment by Adam-it** [comment]
@dgirase  are you still working on this? If the PR is ready for review please click on the 'Ready for review' button to let us know

**Comment by Adam-it** [comment]
@dgirase were you able to build the solution? I tried to test it but I wasn't able to create the package
image

**Comment by dgirase** [comment]
I have resolved the issue of package-solution

**Review by Adam-it — APPROVED** [review]
@dgirase Awesome work 👏👏👏👏
I managed to build it and test it on my tenant
I don't have anything to add from my side.
Perfect 👍. You Rock 🤩

**Pull request merged**